Update README.md (#36122)
fleetdm/fleet: handbook/sales/README.md

5. If there are still unanswered questions, the AE will route the issue to the relevant department. Once complete, the team will leave a comment in the issue `@`mentioning the AE with the requested information. Any questions answered by other departments will be added to the handbook by the AE.

> ### Update Salesforce “Attributed to” field in opportunity page

Reviewer: ```suggestion
### Update Salesforce “Attributed to” field on an opportunity
```

The “Attributed to” field in Salesforce opportunity page will be used to track where each new business opportunity created can be attributed to. This is a temporary solution put in place until we have built-out the process that would automate how we track attribution, and measure ROI associated with our program spend.  

The “Attributed to” field is associated with a Salesforce user. The opportunity owner will have to determine the correct user to add to the field when the opportunity is created. The rule to determine which user to add to the field:

- Inbound: For all opportunities created based on inbound leads (e.g. digital ads, brand, events, GitOps workshops, organic web traffic, etc), update the field with CMO’s name

> - Outbound-AE: If the opportunity was created as a result of the AE’s outbound efforts, update the field with the CCO’s name

Reviewer: ```suggestion
- Outbound-AE: If the opportunity was created as a result of the AE’s outbound efforts, update the field with the AE's name.
```
